Bottom: 0ead4914d7221f66b583b6d9cfbd6176eeef26f8
Top:    d4302d5d9dc7f8c228a3d916a0aadd3c80c68674
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-08-19 16:05:17 +0100

Refresh of play-with-perspective

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index 99bf0f5..e6cc7a3 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -2752,8 +2752,8 @@ ghid_unproject_to_z_plane (int ex, int ey, Coord pcb_z, Coord *pcb_x, Coord *pcb
 //  if (fvx < (double)-G_MAXINT/5.) {fvx = (double)-G_MAXINT/5.; printf ("fvx underflow clamped\n"); }
 //  if (fvy < (double)-G_MAXINT/5.) {fvy = (double)-G_MAXINT/5.; printf ("fvy underflow clamped\n"); }
 
-  *pcb_x = (int)fvx;
-  *pcb_ = (int)fvy;
+  *pcb_x = (Coord)fvx;
+  *pcb_y = (Coord)fvy;
 
   {
     /* Reproject the computed board plane coordinates to eye space */
